docs/how-to/h-enable-monitoring.md

```diff
@@ -0,0 +1,215 @@
+# Enable Monitoring
+
+The Charmed Spark solution comes with the [spark-metrics](https://github.com/banzaicloud/spark-metrics) exporter.
+This exporter is designed to push metrics to the [prometheus pushgateway](https://github.com/prometheus/pushgateway), that is integrated with the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack). 
+
+
+To enable monitor with Charmed Spark just follow this guide.
+
+
+As a prerequisite, you need to have Juju 3 installed with a MicroK8s controller bootstrapped. This can be done following this [tutorial](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s).
+
+
+As a first step, start by deploying cos-lite bundle in a Kubernetes environment with Juju.
+
+```shell
+juju add-model cos
+juju switch cos
+juju deploy cos-lite --trust
+
+```
+You can monitor the status of the deployment with the following command: 
+
+```shell
+watch --color juju status --color --relations
+```
+
+When everything is ready the output of previous command should be something similar:
+```shell
```

Comment:
Since it is an how-to, outputs are not required. I'm saying this also because it would be a bit of a mess to try to keep this output consistent, making it harder to maintain the documentation